app/Http/Controllers/API/v1/NotificationsController.php

```diff
@@ -176,6 +172,6 @@ public function markAsUnread(string $notificationId): JsonResponse {
      */
     public function markAllAsRead(): JsonResponse {
         Auth::user()->unreadNotifications->markAsRead();
-        return $this->sendResponse(['message' => __('notifications.readAll.success')]);
+        return $this->sendResponse();
```

Comment:
This may break our frontend. Please double check.

Reply:
<img width="424" alt="Screenshot 2023-11-12 at 14 37 44" src="https://github.com/Traewelling/traewelling/assets/1267894/5911dde6-d1d2-4869-b774-613e57689546">
oops